apps/spruce/src/pages/adminSettings/index.tsx

```diff
@@ -50,6 +50,29 @@ const AdminSettingsPage: React.FC = () => {
                 Announcements
               </SideNavItem>
             </SideNavGroup>
+            <SideNavGroup header="Feature Flags">
+              <SideNavItem
+                as={Link}
+                data-cy="navitem-admin-feature-flags"
+                to="/admin-settings#services"
+              >
+                Services
+              </SideNavItem>
+              <SideNavItem
+                as={Link}
+                data-cy="navitem-admin-notifications"
+                to="/admin-settings#notifications"
+              >
+                Notifications
+              </SideNavItem>
+              <SideNavItem
+                as={Link}
+                data-cy="navitem-admin-batch-jobs"
+                to="/admin-settings#batch-jobs"
```

Comment:
nit. you can use the new function for `getAdminSettingsRoute` in the `to` properties here